diem-unified-charter.md

```diff
@@ -46,12 +46,10 @@ first milestone of the Working Group.
 - Define discovery of Digital Emblems for Digital Assets
 - Define discovery of Digital Emblems for Physical Assets
 
-### BoF v2 discussion TODO: should this WG adopt both digital and physical assets? 
+# Deliverables
+The following are proposed deliverables and their target status:
 
-It is not typical for IETF work to address the expectations of entities that are
```

Comment:
I'm fine with everything except the removal of this text. I'm working on a re-phrase based on the previous list discussion; we should handle this in a different PR.

Reply:
My perception of the list discussion and the meeting with Eric, Warren, Orie, and Russ indicated that we don't need to have such text in the charter. We expect those interested entities to participate in the standards process.